src/Contracts/Google2FA.php

```diff
@@ -82,11 +82,24 @@ public function removeInvalidChars($string);
 	/**
 	 * Creates a Google QR code url.
 	 *
-	 * @param $company
-	 * @param $holder
-	 * @param $secret
+	 * @param string $company
+	 * @param string $holder
+	 * @param string $secret
+	 * @param integer $size
 	 * @return string
 	 */
-	public function getQRCodeGoogleUrl($company, $holder, $secret);
+	public function getQRCodeGoogleUrl($company, $holder, $secret, $size = 200);
```

Comment:
Indeed, but this is a 0.x release, so BC breaks are possible by tagging a new minor (0.8.0 in this case). I'll update the description to clarify this.